tools/api-markdown-documenter/src/api-item-transforms/helpers/Helpers.ts

```diff
@@ -393,23 +391,32 @@ export function createBreadcrumbParagraph(
 	apiItem: ApiItem,
 	config: ApiItemTransformationConfiguration,
 ): ParagraphNode {
-	// Get ordered ancestry of document items
-	const ancestry = getAncestralHierarchy(apiItem, (hierarchyItem) =>
```

Comment:
This helper was a bit too general and only used in 2 places. I removed it and inlined the logic in the appropriate places.